marlowe-playground-client/src/StaticAnalysis/StaticTools.purs

```diff
@@ -289,6 +322,10 @@ stepAnalysis problemDef settings isCounterExample rad =
       when thereAreNewCounterExamples refreshEditor
       pure result
   where
+  -- FIXME: Refresh editor seems to be updating the simulation slot, probably this is like this
+  --        from a previous use case. At the moment we are not running the analysis from the simulator
+  --        so I don't think this is needed anymore. Confirm before delete.
+  -- mContent <- query _simulatorEditorSlot unit (Monaco.GetText identity)
+  -- for_ mContent (\content -> void $ query _simulatorEditorSlot unit $ Monaco.SetText content unit)
   refreshEditor = do
-    mContent <- query _simulatorEditorSlot unit (Monaco.GetText identity)
-    for_ mContent (\content -> void $ query _simulatorEditorSlot unit $ Monaco.SetText content unit)
+    pure unit
```

Comment:
indeed, we don't use the simulation slot for anything to do with static analysis any more, `refreshEditor` can be removed

Reply:
removed